operators/hpe-csi-operator/deploy/crd.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: hpecsidrivers.storage.hpe.com
+spec:
+  group: storage.hpe.com
+  names:
+    kind: HPECSIDriver
```

Comment:
Is this object necessary?  We already have a CSIDriver object as part of registration within k8s.